iosHyperskillApp/iosHyperskillApp/Sources/Modules/Track/Views/DiscoverNext/TopicToDiscoverNextButtonView.swift

```diff
@@ -98,21 +100,20 @@ struct TopicToDiscoverNextButtonView: View {
     @ViewBuilder
     private var learnNextBadge: some View {
         if isLearnNext {
-            GeometryReader { geometry in
-                Text(Strings.Track.TopicsToDiscoverNext.learnNextBadge)
-                    .font(.caption)
-                    .foregroundColor(Color(ColorPalette.primary))
-                    .padding(.horizontal, LayoutInsets.smallInset)
-                    .padding(.vertical, appearance.learnNextBadgeVerticalPadding)
-                    .background(Color(ColorPalette.surface))
-                    .addBorder(color: Color(ColorPalette.primaryAlpha38))
-                    .offset(
-                        CGSize(
-                            width: LayoutInsets.defaultInset,
-                            height: geometry.size.height * appearance.learnNextBadgeVerticalOffsetRatio
-                        )
+            Text(Strings.Track.TopicsToDiscoverNext.learnNextBadge)
```

Comment:
Cool without 2 `GeometryReader` as a suggested previously 👍

Reply:
Thank you for the idea!